core/plugins/org.fusesource.ide.camel.validation/src/org/fusesource/ide/camel/validation/diagram/BasicNodeValidator.java

```diff
@@ -230,8 +209,7 @@ private IMarker createOrReuseMarker(IResource resource, AbstractCamelModelElemen
 	 * @throws CoreException
 	 */
 	private IMarker createMarker(IResource resource, final String message, final int severity, Map<String, Object> attributesForPosition) throws CoreException {
-		IMarker res;
-		res = resource.createMarker(IFuseMarker.MARKER_TYPE);
+		IMarker res = resource.createMarker(IFuseMarker.MARKER_TYPE);
```

Comment:
Overall though I think this looks great -- centralizes a few things around marker management and validation testing, which is awesome.